src/libZilliqa/Zilliqa.cpp

```diff
@@ -89,7 +89,7 @@ Zilliqa::Zilliqa(const std::pair<PrivKey, PubKey>& key, const Peer& peer,
     {
     case SyncType::NO_SYNC:
         LOG_GENERAL(INFO, "No Sync Needed");
-        Whitelist::GetInstance().UpdateDSWhitelist();
+        Whitelist::GetInstance().Init();
```

Comment:
How about calling Init() inside Whitelist constructor as default private IP exclusion list?
Thus the user do not need to call Init() by themselves.

Reply:
This implementation is fine for now